community/curlpp/build

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh -e
+
+mkdir build
+cd build
+cmake -DCMAKE_INSTALL_PREFIX='/usr' -DCMAKE_INSTALL_LIBDIR='/usr/lib' ../
+make
+make DESTDIR="$1" install
```

Comment:
This isn't how we structure cmake based builds.

See: https://k1ss.org/style (0800)

```sh
export DESTDIR="$1"                                 
                                                   
cmake -B build \                                    
   -DCMAKE_INSTALL_PREFIX=/usr \                   
   -DFLAG=1                                        
                                                   
cmake --build   build                               
cmake --install build
```

The benefit of this syntax is that the user has the choice between Make or Ninja
for the build itself (configurable via CMAKE_GENERATOR). This is a feature of
cmake itself and nothing special to KISS.

Reply:
Very sorry about that, got a bit confused there. I have updated my fork with what I think is the correct syntax.